engine/src/AnimatedSpritesUpdater.cpp

```diff
@@ -0,0 +1,25 @@
+//
+// Created by tdancois on 8/4/16.
+//
+
+#include "AnimatedSpritesUpdater.h"
+
+AnimatedSpritesUpdater *AnimatedSpritesUpdater::instance;
+
+AnimatedSpritesUpdater::AnimatedSpritesUpdater() {}
+
+AnimatedSpritesUpdater *AnimatedSpritesUpdater::getInstance() {
```

Comment:
Well I don't really get the difference. The check `if(!instant)` is only for the first call of `getInstance`.